sdccc/src/main/java/com/draeger/medical/sdccc/messages/mapping/MessageContent.java

```diff
@@ -75,7 +75,7 @@ public class MessageContent {
     private String uuid;
     private boolean isSOAP;
 
-    @Column(name = "sender", length = 15)
+    @Column(name = "sender")
```

Comment:
Unnecessary as it currently is, replace 

> Column(name = "sender") 

with 

> Column(nullable = true)

Reply:
Done.